ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsFetch.scala

```diff
@@ -56,19 +57,21 @@ private class ContractsFetch(
     getActiveContracts: LedgerClientJwt.GetActiveContracts,
     getCreatesAndArchivesSince: LedgerClientJwt.GetCreatesAndArchivesSince,
     getTermination: LedgerClientJwt.GetTermination,
-)(implicit dblog: doobie.LogHandler, sjd: SupportedJdbcDriver)
-    extends StrictLogging {
+)(implicit dblog: doobie.LogHandler, sjd: SupportedJdbcDriver) {
 
   import ContractsFetch._
   import sjd.retrySqlStates
 
+  private[this] def logger = ContextualizedLogger.get(getClass)
```

Comment:
```suggestion
  private[this] val logger = ContextualizedLogger.get(getClass)
```
Any reason to not make it a `val`?

Reply:
This is exactly what I was wondering too a couple hours ago. Thanks for picking this one up, you're right!